internal/fullnode/genesis.go

```diff
@@ -14,8 +14,10 @@ var (
 	scriptUseInitGenesis string
 )
 
-const genesisScriptWrapper = `if [ -f "$GENESIS_FILE" ]; then
-	echo "Genesis file $GENESIS_FILE already exists; skipping initialization."
+const genesisScriptWrapper = `ls $DATA_DIR/*.db 1> /dev/null 2>&1
+DB_INIT=$?
+if [ $DB_INIT -eq 0 ]; then
```

Comment:
This was the approach in the legacy deployments. We know that `ls` exists in the `infra-toolkit` image, so this is safe.

Reply:
I'm being pedantic but I may revert this. I can't tell from the PR description, but I think the original code here was working fine. 

There are areas the operator init-flow that are purposefully different what our legacy deployments do. In this case, I feel it's clearer to check for the existence of the genesis file vs. the existence of `*.db` files. It assumes genesis + dbs must be mutually inclusive which I think is correct. However, if I'm a newcomer looking at this code, it may cause confusion since we're interested in the genesis file in this context. 

Additionally, I did not use `ls` purposefully because of this comment from Mark: https://github.com/strangelove-ventures/cosmos-operator/pull/57#discussion_r949659111

And finally, another benefit is we remove a dependency on `ls` even though it's likely never to be absent from infra-toolkit. 

The other init-flow differences off top of my head are:
* We use `emptyDir` instead of polluting the PVC with temporary files that only init-flow cares about.
* We use `securityContext` to prevent the `chown` step. 

In any case, I'm glad you all were able to work it out and get it fixed!

Reply:
> I think the original code here was working fine.

Noble was unable to start. It would never download the genesis file. The first init container `chain-init` would create a default genesis file, log the error to remove it, then the `genesis-init` would always log `"Genesis file $GENESIS_FILE already exists; skipping initialization."`

> And finally, another benefit is we remove a dependency on `ls` even though it's likely never to be absent from infra-toolkit.

I would argue a dependency on `ls` in an image that we control is easier to maintain than a dependency on `rm` in an image that we may not maintain.

> I'm being pedantic but I may revert this.

I am not sure why it would need to be reverted. An update to use `find` or similar would be more logical than `ls`, but we know that `ls` works fine for this purpose. Alternatively, we could do the initial chain init command in a tmp dir like we do if it's already initialized, and only copy in the config files, that way the genesis file will not be populated, and the previous check for the genesis file existence would work.

Reply:
> Noble was unable to start. It would never download the genesis file. The first init container chain-init would create a default genesis file, log the error to remove it, then the genesis-init would always log "Genesis file $GENESIS_FILE already exists; skipping initialization."

I finally put those pieces together. But thanks for the explanation!

> Alternatively, we could do the initial chain init command in a tmp dir like we do if it's already initialized, and only copy in the config files, that way the genesis file will not be populated, and the previous check for the genesis file existence would work.

Definitely worth consideration. I had another idea to explore where the init-flow is handled by an operator subcommand and all this nuance can be handled in Go code vs. bash. I may explore that later, but not priority in the near future. 

I think this is a good change now and I'll close my PR reverting the changes.